packages/api/src/routes/nfts-store.js

```diff
@@ -66,13 +67,14 @@ export async function nftStore(event, ctx) {
   const size = totalSize(bs)
   const structure = 'Complete'
   const carBytes = await exportToCar(rootCid, bs)
+  const car = new Blob([carBytes])
```

Comment:
won't this make a copy of `carBytes`? I think this is what we were trying to avoid with this: https://github.com/nftstorage/nft.storage/pull/2551/files#diff-fd6c95e488f72dd7865113d9b80c361107edfcff5f5161c87fd35cbaf321a1edL185